reconcile/utils/terrascript_aws_client.py

```diff
@@ -2255,7 +2277,54 @@ def populate_tf_resource_sqs(self, spec):
             tf_resources.append(tf_resource)
 
         self.add_resources(account, tf_resources)
+    
+    def populate_tf_resource_sns(self, spec):
+        account = spec.provisioner_name
+        identifier = spec.identifier
+        common_values = self.init_values(spec)
+        output_prefix = spec.output_prefix
+        policy = common_values.get("inline_policy")
+        region = common_values.get("region") or self.default_regions.get(account)
+        
+        values = {}
+        if 'fifo_topic' in common_values.keys():
```

Comment:
What about replacing this with:

```python
common_values.get("fifo_topic", False)
```
That'd also set the value to False if tthat key is missing?